### ISSUE TITLE
libCBLForestDBStorage.a from 1.2.1-7 build not working with simulator

### ISSUE THREAD

**Issue by pasin** [couchbase/couchbase-lite-ios#1175]
1. The library works fine when running the app on a device but not simulator.
2. I tested on my mac by archiving 'CBL ForestDB Storage' and I can confirm that the result binary is not working. However if I did release build, the binary is working fine. There could be some issue with the target settings.

Error: 
```
WARNING: storageType is 'ForestDB' but no CBL_ForestDBStorage class found; make sure `-ObjC` is in Other Linker Flags in Build Settings
--------
* Version:   1.2.1-7
* Client OS: iOS 9.2, iPhone 6 simulator
* Server:    NA

**Comment by snej** [comment]
I ran Product>Archive on the "CI iOS" scheme, and the ForestDB Storage and CBLJSViewCompiler libraries aren't fat. They only contain ARM binaries:

```
$  pwd
/Users/snej/Library/Developer/Xcode/Archives/2016-03-21/CI iOS 3-21-16, 11.46 AM.xcarchive/Products/usr/local/lib
$  file *
libCBLForestDBStorage.a: Mach-O universal binary with 2 architectures
libCBLForestDBStorage.a (for architecture armv7):	current ar archive random library
libCBLForestDBStorage.a (for architecture arm64):	current ar archive random library
libCBLJSViewCompiler.a:  Mach-O universal binary with 2 architectures
libCBLJSViewCompiler.a (for architecture armv7):	current ar archive random library
libCBLJSViewCompiler.a (for architecture arm64):	current ar archive random library
```

**Comment by snej** [comment]
Fixed by adding a new one-line script build phase to the target, to copy the fat library to the $CONFIGURATION_BUILD_DIR. Did the same for the CBLJSViewCompiler target, which had the same problem.

**Comment by michaelkwok** [comment]
Build archive failure.  There was no meaningful error log.  I removed your changes in the copy step and it passed.  Can you take a look?  Thanks.

**Comment by snej** [comment]
Found the problem — it only failed on a clean build, and I hadn't been doing clean builds. I've pushed a better fix now.

**Comment by michaelkwok** [comment]
@pasin The fixed is in build 1.2.1-9.  Can you verify the simulator issue is fixed now?  Thanks!

**Comment by snej** [comment]
There's nothing [there](http://latestbuilds.hq.couchbase.com/couchbase-lite-ios/release/1.2.1/ios/1.2.1-9/) but a `changes.log` file — no archives. :(
